Fitness/Networking/NetworkManager.swift

```diff
@@ -31,244 +32,152 @@ struct NetworkManager {
             completion(gyms)
         }
     }
-    
+
     func getGymNames(completion: @escaping ([GymNameId]) -> Void) {
-        apollo.fetch(query: AllGymsQuery()){ (result, error) in
+        apollo.fetch(query: AllGymsQuery()) { (result, error) in
             // implement
             var gyms: [GymNameId] = []
-            
+
             for gym in result?.data?.gyms ?? [] {
                 guard let gym = gym else {
                     continue
                 }
-                
-                gyms.append(GymNameId(name: gym.name ?? "", id: gym.id ?? "") )
+
+                gyms.append(GymNameId(name: gym.name, id: gym.id))
             }
-            
+
             completion(gyms)
         }
     }
 
     func getGym(gymId: String, completion: @escaping (Gym) -> Void) {
-//        provider.request(.gym(gymId: gymId)) { result in
-//            switch result {
-//            case let .success(response):
-//                do {
-//                    let gymData = try JSONDecoder().decode(GymRootData.self, from: response.data)
-//                    let gym = gymData.data
-//                    completion(gym)
-//                } catch let err {
-//                    print(err)
-//                }
-//            case let .failure(error):
-//                print(error)
-//            }
-//        }
     }
 
+    func getGymClassesForDate(date: String, completion: @escaping ([GymClassInstance]) -> Void) {
+        apollo.fetch(query: TodaysClassesQuery(date: date)) { result, error in
+            guard let data = result?.data, let classes = data.classes else { return }
+            var gymClassInstances: [GymClassInstance] = []
+            for gymClassData in classes {
+                guard let gymClassData = gymClassData, let imageUrl = URL(string: gymClassData.imageUrl) else { continue }
+                let instructor = gymClassData.instructor
+                let startTime = gymClassData.startTime ?? "12:00:00"
+                let endTime = gymClassData.endTime ?? "12:00:00"
+                let isCancelled = gymClassData.isCancelled
+                let gymId = gymClassData.gymId ?? ""
+                let location = gymClassData.location
+                let classDescription = gymClassData.details.description
+                let className = gymClassData.details.name
+                let start = Date.convertTimeStringToDate(time: startTime)
```

Comment:
I feel like `convertTimeStringToDate` should take in the queried `date` too, or else the day in the start/end times will always be today, even if we're searching for tmr (could make adding stuff to calendar kinda useless)